apps/scoutgame/components/common/DraftDeveloperInfoModal/components/DraftDeveloperBidForm.tsx

```diff
@@ -72,11 +75,13 @@ function DraftDeveloperBidFormComponent({
 
   // Fetch ETH and LINK prices from CoinGecko
   const { data: prices, isLoading: isLoadingPrices } = useSWR('token-prices', async () => {
-    const response = await fetch('https://api.coingecko.com/api/v3/simple/price?ids=ethereum,talent&vs_currencies=usd');
+    const response = await fetch(
```

Comment:
we still need to replace coingecko with decent's response right?

Reply:
correct but decent is not giving the exchange rate rn. Its 0